Gemfile

```diff
@@ -3,8 +3,8 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
-gem 'solidus', github: 'solidusio/solidus', branch: branch
+branch = ENV.fetch('SOLIDUS_BRANCH', 'v2.10-penrose')
```

Comment:
I don't think we need to do this? It should work just fine pointing at solidus master.